bazel/grpc_build_system.bzl

```diff
@@ -128,6 +128,11 @@ def grpc_cc_library(
         linkstatic = linkstatic,
     )
 
+def grpc_cc_library_xds(
```

Comment:
Please add a comment here saying that once we can depend on the xDS protos internally, this should go away, and all build rules using it should be changed back to use `grpc_cc_library`.

Reply:
Done.